assignments/arrays.js

```diff
@@ -63,32 +63,66 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a 2011 Jeep Wrangler` );
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
+let lastCar = inventory.car_make[inventory.length - 1] + inventory.car_model[inventory.length - 1]
+
+console.log(lastCar)
+
+
+
+
 console.log();
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+
+for (i = 0; i < inventory.length; ++i){
+    carModels.push(inventory.car_model[i]);
+    carModels.sort();
```

Comment:
This will work, but you are unnecessarily sorting the list of carModels during every iteration of this loop. You could instead run sort once after the loop is completed.